resources/js/Components/Form/CampaignCreateForm.tsx

```diff
@@ -108,9 +115,21 @@ const CampaignCreateForm = ({
             </Grid>
             <Grid container spacing={4}>
                 <Grid item xs={4}>
-                    <Button onClick={() => onCloseDrawer && onCloseDrawer()} fullWidth>
-                        Cancel
-                    </Button>
+                    {type === 'Create' ? (
```

Comment:
dont use ternary for rendering 2 different buttons, use ternary in the onClick function instead